.licenseignore

```diff
@@ -39,7 +39,6 @@ Sources/CNIOLLHTTP/c_nio_llhttp.c
 Sources/CNIOLLHTTP/include/c_nio_llhttp.h
 Sources/CNIOSHA1/c_nio_sha1.c
 Sources/CNIOSHA1/include/CNIOSHA1.h
-dev/alloc-limits-from-test-output
```

Comment:
This script _is_ used, I updated it in #2987.

Reply:
huh, ok, will revert